nuxeo-services/nuxeo-platform-convert/src/test/java/org/nuxeo/ecm/platform/convert/tests/TestAnyToPDFConverters.java

```diff
@@ -64,7 +71,7 @@ protected String doTestPDFConverter(String srcMT, String fileName, boolean pdfa,
 
         Map<String, Serializable> parameters = new HashMap<>();
         if (pdfa) {
```

Comment:
I don't like that there are now two pdfa-based arguments in the method, it's incoherent. Just change the existing `pdfa` parameter to your `Serializable` and change this test to `if (pdfa != null)`.